### ISSUE TITLE
Errors since update to latest version (6.0.0)

### ISSUE THREAD

**Issue by mhosman** [urish/ngx-moment#272]
- Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)
- Error: export 'TimeAgoPipe' (imported as 'i26') was not found in 'ngx-moment' (module has no exports)
- Error: export 'FromUtcPipe' (imported as 'i26') was not found in 'ngx-moment' (module has no exports)

And so on....

**Comment by ct5845** [comment]
Similiar issues
 

`Error: Symbol TimeAgoPipe declared in $/node_modules/ngx-moment/time-ago.pipe.d.ts is not exported from ngx-moment/moment.module`

**Comment by CynanX** [comment]
I've the same..

**Comment by ct5845** [comment]
Might be irrelevant, but when trying to build the master branch i'm getting this error;

```
Building Angular Package

------------------------------------------------------------------------------
Building entry point 'ngx-moment'
------------------------------------------------------------------------------
× Compiling with Angular sources in Ivy partial compilation mode.
node_modules/@types/eslint/index.d.ts:450:42 - error TS2724: '"C:/Users/***/Documents/GitHub/ngx-moment/node_modules/@types/estree/index"' has no exported member named 'ChainExpression'. Did you mean 'ThisExpression'?

450         ChainExpression?: ((node: ESTree.ChainExpression & NodeParentExtension) => void) | undefined;
                                             ~~~~~~~~~~~~~~~
node_modules/@types/eslint/index.d.ts:473:43 - error TS2694: Namespace '"C:/Users/***/Documents/GitHub/ngx-moment/node_modules/@types/estree/index"' has no exported member 'ImportExpression'.

473         ImportExpression?: ((node: ESTree.ImportExpression & NodeParentExtension) => void) | undefined;
                                    
```

**Comment by aleperfetti81** [comment]
Hi have the same error message:
Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)

**Comment by urish** [comment]
Sorry, but I don't have the bandwidth to look into this right now. If anyone is up to it, I'd love to see a pull request. Otherwise I'll just revert the change and go back to v5.0.0

**Comment by Rambou** [comment]
@urish the v6 has already been publish on npmjs https://www.npmjs.com/package/ngx-moment it needs to be fixed now

**Comment by tarlepp** [comment]
Yep, I noticed the same after updating - `Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)`

**Comment by marvin-w** [comment]
I have the same error

**Comment by RodolfoXDR** [comment]
Same error here!
`./src/app/app.module.ts:69:12-32 - Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)`

`./src/app/app.module.ts:92:64-79 - Error: export 'MomentModule' (imported as 'i4') was not found in 'ngx-moment' (module has no exports)`

**Comment by tarlepp** [comment]
hmm, no cannot see that `6.0.0` on releases page - https://github.com/urish/ngx-moment/releases/ - but npmjs.com is showing that `6.0.0` release - https://www.npmjs.com/package/ngx-moment

@urish was that a mistake version bump or are we really facing real issue here?

**Comment by urish** [comment]
@tarlepp real issue

**Comment by lacroixdavid1** [comment]
Same issue here after updating to 6.0.0.

```

./projects/network/src/app/routes/offer/offer/offer-view/offer-view.module.ts:78:8-20 - Error: export 'DurationPipe' (imported as 'DurationPipe') was not found in 'ngx-moment' (module has no exports)

./projects/network/src/app/routes/offer/offer/offer-view/offer-view.module.ts:83:12-24 - Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)

./projects/network/src/app/routes/offer/offer/offer-view/offer-view.module.ts:160:8-20 - Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)

```

**Comment by Koslun** [comment]
Would appear that all exports from the package is broken? Hoping for some hero to fix this.

This is our errors:

```
 ./SOME_PATH - Error: export 'DateFormatPipe' (imported as 'DateFormatPipe') was not found in 'ngx-moment' (module has no exports)
../SOME_PATH - Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)
```

**Comment by tobigit** [comment]
Same error here:
`Error: export 'MomentModule' (imported as 'MomentModule') was not found in 'ngx-moment' (module has no exports)`

**Comment by tarlepp** [comment]
@urish is there some estimate when this is fixed? or is there something that we could help you with this issue?

**Comment by urish** [comment]
@tarlepp a pull request with a fix would be appreciated. I don't currently have the bandwidth to dig into why the tooling doesn't package the code correctly

**Comment by tarlepp** [comment]
looks like this repository doesn't have that `6.0.0` tag anymore but it exists on `https://www.npmjs.com/package/ngx-moment` - should that be also reverted?

**Comment by CynanX** [comment]
Hi. So I just came to take a look at this and see if I could fix it. I pulled down the master branch, built it locally, pointed to it in my local project and it works fine. Perhaps the issue is a bad deployment to the NPM repository only?

**Comment by tarlepp** [comment]
or maybe those two commits after `6.0.0` just fixed this already - https://github.com/urish/ngx-moment/commits/master ?

**Comment by ETMitch21** [comment]
I'm experiencing this issue still and I just installed to a project.

**Comment by Delagen** [comment]
Resolved in #273

**Comment by Delagen** [comment]
@urish Sad by this occured https://github.com/urish/ngx-moment/issues/245. I open PR soon.
